packages/react/src/components/Menu/MenuItem.tsx

```diff
@@ -214,7 +215,8 @@ class MenuItem extends AutoControlledComponent<WithAsProp<MenuItemProps>, MenuIt
     } = this.props
     const { menuOpen } = this.state
 
-    const indicatorWithDefaults = indicator === undefined ? {} : indicator
+    const defaultIndicator = { name: vertical ? 'stardust-arrow-end' : 'stardust-arrow-down' }
+    const indicatorWithDefaults = indicator === undefined ? defaultIndicator : indicator
```

Comment:
Are these changes required? Can we revert them?

Reply:
It is required, because Type `{}` is not assignable to type `Props<IconProps>`.